test/linter/test.sh

```diff
@@ -35,6 +35,11 @@ if [ $? -ne 0 ]; then
   exit 1
 fi
 
+helm lint charts/opa --strict --set useHttps=false
+if [ $? -ne 0 ]; then
+  exit 1
+fi
```

Comment:
[nit] If we put a `set -e` on top, we could reduce the file to
```bash
helm lint charts/opa --strict
helm lint charts/opa --strict --set mgmt.enabled=true
helm lint charts/opa --strict --set sar.enabled=true
# etc
```
...since any non-0 exit code would cause the script to fail. But it's just cleanup, no function change.

Reply:
By councidence, those tests are currently validating only _positive cases_, i.e. when we expect linting to pass, returning exit code 0.
But theintrntion is to add _negative cases_ where we will expect linting to fail, ie return non zero exit code.
Setting `set -e` would make the entire script fail immediately, as soon as any lint fail, so we cant test _negative case_.

Thats why `set -e` is deliberately omitted.